workbench-core/accounts/src/services/costCenterService.ts

```diff
@@ -131,7 +132,20 @@ export default class CostCenterService {
 
   public async create(request: CreateCostCenterRequest): Promise<CostCenter> {
     const id = uuidWithLowercasePrefix(resourceTypeToKey.costCenter);
-    const account = await this._getAccount(request.accountId);
+    let account: Account;
+
+    const accountService = new AccountService(this._dynamoDbService);
```

Comment:
Why take this out of the helper method?

Reply:
the helper method was only a couple lines long and hid another throw that could happen. Since it doesn't need to be shared, I felt it would make the code easier to reason about because you can see what failure scenarios you need to handle.